manifest.md

```diff
@@ -0,0 +1,261 @@
+<!--[metadata]>
++++
+draft = true
++++
+<![end-metadata]-->
+
+# Image Manifest
+
+The first goal is content-addressable images, by supporting an image model where the image's configuration can be hashed to generate an ID for the image and its components.
+The second goal is to allow multi-architecture images, through a "fat manifest" which references image manifests for platform-specific versions of an image.
+The third goal is to be translatable to the [OpenContainers/runtime-spec](https://github.com/opencontainers/runtime-spec)
+
+# Media Types
+
+The following media types are used by the manifest formats described here, and the resources they reference:
+
+- `application/vnd.oci.image.manifest.v1+json`: Image manifest format
+- `application/vnd.oci.image.manifest.list.v1+json`: Manifest list, aka "fat manifest"
+- `application/vnd.oci.image.serialization.v1+json`: Container config JSON
+- `application/vnd.oci.image.rootfs.tar.gzip`: "Layer", as a gzipped tar
```

Comment:
A docker v2.2 image is this JSON manifest and then a collection of tarballs that make up the image layers. We might define a format that puts all of these things into a single file but I don't think there is a ton of utility since you sign this json manifest, and this would create a second signable object in the system.